core/sender/progress_count_deploy_resource.go

```diff
@@ -0,0 +1,26 @@
+package sender
+
+import (
+	pb "github.com/odpf/optimus/api/proto/odpf/optimus/core/v1beta1"
+)
+
+type deployResourceProgressCount struct {
```

Comment:
We don't need the progress, as this is not helpful to the users and this response is tightly coupled with the optimus cli. So, better avoid it.